code/__defines/_macros.dm

```diff
@@ -2,6 +2,18 @@
 #define CLAMP01(x) 		(Clamp(x, 0, 1))
 
 #define span(class, text) ("<span class='[class]'>[text]</span>")
+#define SPAN_NOTICE(X) ("span class='notice'>[X]</span>")
```

Comment:
Why not define these using the span macro we have defined just above?

```suggestion
#define SPAN_NOTICE(X) span("notice", X)
```

Alternatively, you're missing the appropriate opening `<` tags and stuff.